actions/form/class.Instance.php

```diff
@@ -227,4 +230,42 @@ private function isEmptyLabel($element): bool
         return $element instanceof tao_helpers_form_elements_Label
             && empty($element->getRawValue());
     }
+
+    /**
+     * Check if the Solar design is enabled and the postfix has not yet been added
+     *
+     */
+    private function isContainPostfix(string $language): bool
+    {
+        $pattern = '/' . self::LANG_POSTFIX . '$/';
```

Comment:
Can we use `sprintf` here? 
```suggestion
       $pattern = sprintf('/%s$/', self::LANG_POSTFIX);
```